Azure/sonic-buildimage
# [DELL]Fixing orchagent crash in S6100,Z9100

**- What I did**
Removed the 10G ports that are defined in config.bcm but not in port_config.ini. With new breakout changes, syncd tries to remove these ports during bootup and broadcom SAI returns error assuming delete call comes for breakout and the 10G ports don't support breakout.

**- How I did it**
Removed configs from config.bcm

**- How to verify it**
Load Z9100 and S6100 with these change and verify if there is no orchagent crash in Z9100,S6100

**- Description for the changelog**

Removed unused 10G port related configurations from config.bcm to avoid orchagent crash during bootup

**- A picture of a cute animal (not mandatory but encouraged)**
